phylogenetic/Snakefile

```diff
@@ -1,5 +1,4 @@
-if not config:
-    configfile: "config/config_zika.yaml"
+configfile: "config/config_zika.yaml"
```

Comment:
_non-blocking_ 

With this change to always use the default config YAML, the config access throughout the workflow can be updated from `config.get("strain_id_field", "strain")` to `config["strain_id_field"]` without the hard-coded default value. Embedding default values throughout the Snakemake workflow has potential to be confusing in the future.